.jenkins/Jenkinsfile

```diff
@@ -82,7 +82,7 @@ def ACCContainerTest(String label, String version) {
                         ninja -v
                         ctest --output-on-failure
                        """
-                dcap.ContainerRun("${DOCKER_REGISTRY}/oetools-full-${version}:latest", "clang-8", task, "--cap-add=SYS_PTRACE --device /dev/sgx:/dev/sgx")
+                dcap.ContainerRun("${DOCKER_REGISTRY}/oetools-full-${version}:latest", "clang-10", task, "--cap-add=SYS_PTRACE --device /dev/sgx:/dev/sgx")
```

Comment:
Determine how dcapdockerciregistry.azurecr.io/oetools-full is created. I suspect it is similar to those at oeciteam -- if so then it is better to use the ones directly from oectiteam such as https://hub.docker.com/r/oeciteam/oetools-18.04

If you do you'll need to also mount `/var/run/aesmd`

Repeat for all occurrences